src/Illuminate/Database/Eloquent/Concerns/QueriesRelationships.php

```diff
@@ -222,7 +222,7 @@ public function hasMorph($relation, $types, $operator = '>=', $count = 1, $boole
                         };
                     }
 
-                    $query->where($relation->getMorphType(), '=', (new $type)->getMorphClass())
+                    $query->where($this->query->from.'.'.$relation->getMorphType(), '=', (new $type)->getMorphClass())
```

Comment:
I believe this needs to be `$this->wrapTable($this->query->from)`, there are other such examples if you search for `->wrapTable()`

Reply:
I used the same syntax used in `QueriesRelationships@withCount` method row **363**. Do you suggest to update also that one in the way to standardize?